app/src/org/commcare/android/view/EntityView.java

```diff
@@ -253,19 +260,46 @@ public void onClick(View v) {
         }
     }
 
+    private void addLayoutToRedrawQueue(View layout, String source) {

```

Comment:
not convinced this function is adding value, given that `imagesViewsToRedraw` is a nice descriptive name, but I don't feel that strongly about it